spec/errorutils.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { MobileV3PlayerErrorEvent } from '../src/ts/mobilev3playerapi';
 import { ErrorUtils } from '../src/ts/errorutils';
 import defaultMobileV3ErrorMessageTranslator = ErrorUtils.defaultMobileV3ErrorMessageTranslator;
-import { ErrorEvent } from 'bitmovin-player';
+import { ErrorEvent, PlayerEvent } from 'bitmovin-player';
```

Comment:
why do we need this `PlayerEvent` import? I don't see it used anywhere.

Reply:
We don't, unused import. This was a left over from my tries to properly fix this 😅 
Fixed in 21bd209d